app/javascript/components/hocs/with-fragment.jsx

```diff
@@ -16,6 +23,10 @@ export default function withFragment(WrappedComponent, fragmentObject) {
       };
     }, {});
 
+    if (componentName != null) {
+      console.log('After Data Props: ', fragmentDataProps);
```

Comment:
What does 'After Data Props' mean?

Reply:
This statement was just meant to check the differences of props before and after `fragmentDataProps` was computed.